src/_includes/cwl/additional-arguments-and-parameters/arguments.cwl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env cwl-runner
+#!/usr/bin/env cwltool
```

Comment:
Nope,  not in the `#!` lines 😂

Reply:
Replaced with IDE the shebangs, s/#!/usr/bin/env cwltool/#!/usr/bin/env cwl-runner :+1: